tmt/steps/report/reportportal.py

```diff
@@ -147,8 +149,13 @@ class ReportReportPortal(tmt.steps.report.ReportPlugin[ReportReportPortalData]):
       resulting in one or more launches.
     * suite-per-plan with reported structure 'launch > suite > test'
       resulting in one launch only, and one or more suites within.
-      (Recommended to define launch-name and launch-description in
-      addition)
+      (Recommended to define launch-name and launch-description in addition)
+
+    To override the default reportportal API version (v1):
+
+    .. code-block:: bash
+
+        export TMT_PLUGIN_REPORT_REPORTPORTAL_API_VERSION="v2"
```

Comment:
Is this bit necessary? It focuses on CLI only without telling it to the user. I may easily override RP API in fmf file as well, right? I can easily override many other keys too, yet they are not mentioned. Written like this, it seems like an ad-hoc addition, and I for one would remove it, because on line 116 I already see a paragraph describing how one can override keys on the command line.